packages/cli/src/Server.ts

```diff
@@ -479,6 +480,13 @@ class Server extends AbstractServer {
 			this.app.use(`/${this.restEndpoint}/ldap`, ldapController);
 		}
 
+		// ----------------------------------------
+		// SAML
+		// ----------------------------------------
+		if (isSamlEnabled()) {
```

Comment:
I think we'll have to rethink this check, because if n8n is started without SAML enabled and I then upgrade (by purchasing a License) it'd require restarting. Can we always load the controller but have a middleware that causes requests to fail when not available?

Reply:
That or we can simply check within the controller functions

Reply:
good idea, should probably also be done for ldap